docs/support/faq/how-to-use-pouta-for-remote-desktop.md

```diff
@@ -115,21 +121,22 @@ You can start the noVNC client by running
 /var/lib/snapd/snap/bin/novnc --listen 6081 --vnc localhost:5901
 ```
 
-The location of the application might be in a different location based on what
+The location of the application might be in a different based on what
 Linux distribution you are using.
 The `--listen 6081` means on which port the service will be accessed from. The
 `--vnc localhost:5901` means on what port it is expecting the vncserver to be 
 accessed from. You can exit out of the noVNC session by `ctrl+c`.
 
 You should now be able to access the noVNC session by going to this link in
-your browser `http://127.0.0.1:2001/vnc.html` . Note that the port number is the
+your browser `http://127.0.0.1:2001/vnc.html` . Note, that the port number is the
 same as the one you used with the ssh-command.
 
-You can see your vncserver sessions by running. Note that with this documentation the NoVNC session will only work ':1'
+You can see your vncserver sessions by running the following command.
 
 ```
 vncserver -list
 ```
+Note that with this documentation the NoVNC session will only work ':1'
```

Comment:
Food for thought, maybe this sentence should be on line `106`?

 ( Also should `Note` be `Note,`? 😄  )